src/pages/[userId]/prospects.js

```diff
@@ -1,15 +1,34 @@
 import { useRouter } from 'next/router';
 import styled from '@emotion/styled';
 import {
-  pipe, join, length, path, prop,
+  pipe, join, length, path, prop, ifElse,
 } from 'ramda';
+import { format } from 'date-fns';
+import add from 'date-fns/add';
 import Grid from '@material-ui/core/Grid';
-import { DesktopOnly, MobileOnly } from 'components/MediaQuery';
+import { DesktopOnly, MobileOnly } from '../../components/MediaQuery';
+import { getProspects } from '../../modules/partners';
+import { getMinisterById } from '../../modules/ministers';
 
 export default function Prospects() {
   const router = useRouter();
   const { userId } = router.query;
+  const {
+    data: prospects,
+    isLoading: isLoadingProspects,
+    error: prospectsError,
```

Comment:
Let's go ahead and display `prospectsError` and `userError` if they exist.

Reply:
Done. I left it with pretty simple messages, but I think it gets the job done